rfc/rfc003-oauth2-authorization.md

```diff
@@ -245,11 +248,17 @@ It MUST be verified that the actor's request conforms to the specified policy an
 
 The **vcs** field is not needed when actor and custodian are the same.
 
-**5.2.1.8. Subject validation**
+**5.2.1.8 Subject validation**
 
 The **sub** field in the JWT MUST be a known organization. It MUST have been registered by the node operator of the authorization server and it MUST be valid at the time indicated by the **iat** field.
 
-#### 5.2.2. Error responses
+**5.2.1.9  Purpose of use**
+
+The `purposeOfUse` field is to define the scope of the access token. It contains a list of policy names. The policy names are defined by Bolts.
+All `purposeOfUse` entries from the accompanying authorization credentials MUST be listed.
+The resource server MUST be able to resolve the policy names from the access token. 
```

Comment:
what does "resolving policy names" mean? Resolving policy names to known Bolt policies? (specify)